### ISSUE TITLE
suprising behavior of iterative route-file loading

### ISSUE THREAD

**Issue by namdre** [DLR-TS/sumo#3593]
when loading vehicles with departures spaced evenly every 10 seconds and option --route-steps 100

in step 0 vehicles departing at 0 and 10 are loaded (only the first vehicle the departs at time > 0)
in step 10 vehicles departing at 10-120 are loaded (only the first vehicle that departs at time > 110)
in step 120 vehicles departing at 130-230 are loaded (only the first vehicle that departs at time > 220)
in step 230 vehicles departing at 240-340 are loaded (only the first vehicle that departs at time > 330)
...

Basically, the first loaded chunk is much smaller than expected.
This is only of interest when trying to access routes via TraCI.
